ToDoApp/App.js

```diff
@@ -8,8 +8,8 @@
  */
 
 import React, {Component} from 'react';
-import {Platform, StyleSheet, Text, View} from 'react-native';
-import {createStackNavigator, createAppContainer} from 'react-navigation';
+import {Platform, StyleSheet, View} from 'react-native';
+import Home from './containers/home/Home';
 
 const instructions = Platform.select({
```

Comment:
please remove this line, its no longer useful

Reply:
I have just changed that, in the next push it will be added.